build_resolvers/lib/src/resolver.dart

```diff
@@ -84,7 +84,12 @@ class AnalyzerResolver implements ReleasableResolver {
   final _seenAssets = new Set<AssetId>();
 
   AnalyzerResolver(DartUriResolver dartUriResolver) {
-    _context.analysisOptions = new AnalysisOptionsImpl()..strongMode = true;
+    _context.analysisOptions = new AnalysisOptionsImpl()
+      ..strongMode = true
+      ..analyzeFunctionBodies = false
+      ..disableCacheFlushing = false
+      ..trackCacheDependencies = false
```

Comment:
Based on the Doc comment this one seems unsafe

https://github.com/dart-lang/sdk/blob/1afe71c08af19cd627bc7bc07495445230f2442b/pkg/analyzer/lib/src/generated/engine.dart#L1309-L1315

@jakemac53 what do you think?

Reply:
Ya, it doesn't sound like its safe to set this to `false`